gate-web/config/gate.yml

```diff
@@ -141,11 +141,15 @@ security:
       clientAuthenticationScheme: query
     resource:
       userInfoUri: https://graph.windows.net/me?api-version=1.6
+      userRoleInfoUri: https://graph.windows.net/me/getMemberGroups?api-version=1.6
```

Comment:
I'd like someone from @spinnaker/microsoft to chime in here to ensure that all Azure users would have roles populated this way.